src/FreeDSx/Snmp/Message/Security/UsmSecurityParameters.php

```diff
@@ -229,9 +229,9 @@ public function toAsn1(): AbstractType
     /**
      * {@inheritdoc}
      */
-    public static function fromAsn1(AbstractType $type)
```

Comment:
Was this change just to make the variable name more explicit?

Reply:
it was because there's another `$type` variable defined in foreach later. That basically conflicts, should be avoided.

Reply:
Ah, ok. Didn't catch that. Makes sense.